### PR TITLE
Feature/process model outputs

### PR DESCRIPTION
**Pull request by ray-millward-tessella**
Updated the API to download model files from blob storage, transform them and add a trigger file. The tile server then restarts when it sees this change. All of the API config files get updated at the same time so the new map is immediately available.

Integrated in the Azurite local blob storage emulator with connection strings for the emulator.

### CODE REVIEW

File: src/API/.envrc.template [hunk 1]
```diff
@@ -7,4 +7,4 @@ export PORT=7000
 export AUTH0_ISSUER_URL='https://dev-326tk4zu.us.auth0.com/'
 export AUTH0_AUDIENCE='https://www.vectoratlas.org'
 export TOKEN_KEY='<token_key>'
-export AZURE_STORAGE_CONNECTION_STRING='<connection_string>'
\ No newline at end of file
+export AZURE_STORAGE_CONNECTION_STRING='DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;'
```
Comment (andykitch) [inline src/API/.envrc.template:10]: Uh-oh :-)

File: src/API/src/models/handlers/blobHandler.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  const blobServiceClient = BlobServiceClient.fromConnectionString(
+    config.get('blobStorageConnectionString'),
+  );
+  const containerClient = blobServiceClient.getContainerClient(
+    'vectoratlas-container',
+  );
+
+  const blobClient = containerClient.getBlockBlobClient(blobLocation);
+
+  await blobClient.downloadToFile(`${BLOB_FOLDER}${modelOutputName}.tif`);
```
Comment (andykitch) [inline src/API/src/models/handlers/blobHandler.ts:17]: Is it always going to be tifs? The modeul upload allows tifs and shp's

File: src/API/src/models/models.resolver.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import { Args, Query, Resolver, Field, ObjectType } from '@nestjs/graphql';
+import { ModelsTransformationService } from './modelsTransformation.service';
+
+export const stringTypeResolver = () => String;
+export const numberTypeResolver = () => Number;
```
Comment (andykitch) [inline src/API/src/models/models.resolver.ts:5]: Feels like these should be in a common place somewhere, we have them in a few resolver files

File: src/API/.envrc.template (hunk 1 above)
Comment (andykitch) [reply src/API/.envrc.template:10]: Oh wait, this is for the local one?

Comment (ray-millward-tessella) [reply src/API/.envrc.template:10]: Yes, quite right - this is only for the emulator and is publicly available on the Microsoft site anyway. There's nothing secret about this value.

File: src/API/src/models/handlers/blobHandler.ts (hunk 2 above)
Comment (ray-millward-tessella) [reply src/API/src/models/handlers/blobHandler.ts:17]: No, but I needed to get it to work for tifs first - I think we need to make a new story for shp files. I've built it so it should be relatively simple to make it work for shape files as well.

File: src/API/src/models/models.resolver.ts (hunk 3 above)
Comment (ray-millward-tessella) [reply src/API/src/models/models.resolver.ts:5]: I think you're right - I'll look at refactoring it.

Comment (ray-millward-tessella) [reply src/API/src/models/models.resolver.ts]: I've now factored out the ones in model.resolver.ts - we might want to pull out others from the db ingest process over time.

**Pull request merged**